src/app/core/issue-management/_services/issue-management.service.ts

```diff
@@ -35,7 +35,31 @@ export class IssueManagementService {
       catchError(e => {
         this.toasterService.pop('error', 'Getting issue list', e.error.message)
         return [];
+      })
+    )
+  }
+
+  public getIssueById(id: string): Observable<Issue> {
+    return this.http.get<Issue>(this.repoEndpoint + this.serviceEndpoint + `/${id}`).pipe(
```

Comment:
Prefer using a single string template over a mix of concatenation and string template